IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SignalSighting/DocPropertySet.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SignalSighting" UniqueId="764b495c-c373-3a7d-3e5f-32e13622b381" ApplicableType="IfcSignal" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
Unsure about this one, but maybe this should also be "RailwaySignalSighting"

Reply:
For this one, I am not sure whether it is needed to change to RailwaySignalSighting. In my understanding, sighting is anyway needed. But if you think road don't have these requirements, I am fine to change to RailwaySignalSighting.

Reply:
I would be surprised if sighting of signals in other domains carries the same significance.